azure-cli/monitor/Initialize-ApplicationInsights.ps1

```diff
@@ -47,8 +47,8 @@ function Initialize-ApplicationInsights {
   }
   else {
     Write-Host " -> Resource exists." -ForegroundColor Cyan
-    $instrumentationKey = ($response | ConvertFrom-Json -AsHashtable).instrumentationKey
```

Comment:
We were not using  `scriptType: pscore` in our yaml pipeline, causing this line to fail.

Just FYI, @lupusbytes

Reply:
7.2 is pscore - also on Windows (we were using scriptType: ps before)

Reply:
`ps` on a Windows agent is not 7.x ? I thought that.

Reply:
well, then we learned something today

Reply:
is `ps` PowerShell 5 then?

Reply:
That is my interpetation, yes:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-cli?view=azure-devops#task-inputs